server/server/src/upload_fridge_tag.rs

```diff
@@ -79,18 +66,13 @@ fn upload_fridge_tag(
         .basic_context()
         .context("Cannot get connection")?;
 
-    let file = form.files.pop().context("Cannot find attached file")?;
-    let file_name = file.file_name.context("Filename is not specified")?;
-
-    let dir = prepare_file_dir(TEMP_FRIDGETAG_FILE_DIR, &settings.server.base_dir)?;
-
-    let new_file_path = dir.join(file_name);
+    let file_service = StaticFileService::new(&settings.server.base_dir)?;
 
-    move_file(file.file.path(), &new_file_path)?;
+    let static_file = file_service.move_temp_file(file, &StaticFileCategory::Temporary, None)?;
```

Comment:
StaticFileCategory::Temporary would mean this file will be cleaned up

Reply:
FYI - clean up is on file download request right now. So if someone was to upload lots of fridge tags and not download any report the clean up wouldn't be triggered but not such a big deal I think...

Reply:
Should probably but in a refactor issue at some point to do file clean up in a background process.

Reply:
Agree, i'll just add it to carry over to not forget to insert this. https://github.com/msupply-foundation/open-msupply/issues/3666